modules/http/http-worker.c

```diff
@@ -160,7 +160,7 @@ _setup_static_options_in_curl(HTTPDestinationWorker *self)
     {
       curl_easy_setopt(self->curl, CURLOPT_FOLLOWLOCATION, 1);
       curl_easy_setopt(self->curl, CURLOPT_POSTREDIR, CURL_REDIR_POST_ALL);
-      curl_easy_setopt(self->curl, CURLOPT_REDIR_PROTOCOLS, CURLPROTO_HTTP | CURLPROTO_HTTPS);
+      curl_easy_setopt(self->curl, CURLOPT_REDIR_PROTOCOLS_STR, CURLPROTO_HTTP | CURLPROTO_HTTPS);
```

Comment:
The new option is available since curl v7.85.0.
We might need to use conditionals here.
https://repology.org/project/curl/versions

Reply:
I added this too, let me see if this compiles.